src/Npgsql/MultiHostConnectorPool.cs

```diff
@@ -178,10 +178,9 @@ static ClusterState GetClusterState(string host, int port, bool ignoreExpiration
                 }
                 catch (Exception ex)
                 {
+                    exceptions.Add(ex);
```

Comment:
Just noting that this means that non-PostgresException are also no longer wrapped by NpgsqlException; so e.g. IOException inside AggregateException will be there directly rather than wrapped. That seems good to me - the point of wrapping NpgsqlException is to allow users to catch Npgsql-related exceptions in their code, and we already allow that by wrapping the AggregateException itself; wrapping the inner exceptions doesn't really help in any way.